### ISSUE TITLE
Bisqify BitcoinNews.com - Increase liquidity

### ISSUE THREAD

**Issue by hyperfree** [bisq-network/growth#281]
TLDR; introduce the BitcoinNews audience to bisq with a full-page-takeover ad campaign and create lifetime bisq users

BitcoinNews is a fast growing new independent news and content organization dedicated to educate and inform about Bitcoin and Lightning.

To keep the operation going and hire more writers we sell advertising space on our website. We work strictly and exclusively with solid bitcoin-only brands.

As the BitcoinNews team is familiar with bisq and uses it for years, we have already qualified that we can accept bisq as an advertising partner. For this reason we would like to propose to the bisq team to collaborate. 

The partnership can entail educational content in form of articles, video, Youtube shorts and tweets.

Demo:
https://youtu.be/pbUk3vlODn8

https://bitcoinnews.com/the-ethics-of-mixing/

If the proposal is of interest, more details about reach and numbers can be provided.
